docs/CONTRIBUTING.rst

```diff
@@ -18,24 +18,72 @@ know like-minded people and make something great together.
 So let's find which contribution option fits you the best and help you
 join the development of LVGL!
 
-Before getting started here are some guidelines to make contribution smoother:
-
-- Be kind and friendly.
-- Be sure to read the relevant part of the documentation before posting a question.
-- Ask questions in the `Forum <https://forum.lvgl.io/>`__ and use
-  `GitHub <https://github.com/lvgl/>`__ for development-related discussions.
-- Always fill out the post or issue templates in the Forum or GitHub (or at least provide equivalent information). It makes
-  understanding your contribution or issue easier and you will get a useful response faster.
-- If possible send an absolute minimal but buildable code example in order to reproduce the issue. Be sure it
-  contains all the required variable declarations, constants, and assets (images, fonts).
-- Use `Markdown <https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet>`__
-  to format your posts. You can learn it in 10 minutes.
-- Speak about one thing in one issue or topic. It makes your post easier to find later for
-  someone with the same question.
-- Give feedback and close the issue or mark the topic as solved if your question is answered.
-- For non-trivial fixes and features, it's better to open an issue first to discuss the
-  details instead of sending a pull request directly.
-- Please read and follow the Coding style guide.
+Ways to contribute
+-------------------
+
+- **Spread the Word**: Share your LVGL experience with friends or on social media to boost its visibility.
+- **Start the LVGL `repository <https://github.com/lvgl/lvgl>`__**
```

Comment:
Maybe it's better to use "Fork" or "Clone" verbs instead of "Start"

Reply:
I meant "Star". :slightly_smiling_face: